vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -0,0 +1,150 @@
+   - Set the `Ingress Host Name` as in Scenario 1 above
+   - Keep the default `DBMS` as `postgresql`
+10. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources.
+
+Once the deployment is complete you can confirm that all routes use https with OpenShift's CA. You will likely still see security warnings in the browser.
+
+See the [Next Steps](#next-steps) below to continue your work with Entando.
+
+## Scenario 4 - PostgreSQL plus self-signed SSL
+This scenario is similar to Scenario 3 but here you'll use a self-signed certificate rather than using OpenShift's Certificate Authority. As a starting point, you can either remove the EntandoApp and Provided Capabilities from the previous scenarios or prepare a new project per steps 1-5 in Scenario 1.
+
+We'll start by creating a self-signed certificate and then prepare the Secrets and ConfigMap to match. There are various ways to create an X.509 self-signed certificate so you can use your preferred mechanism.
+
+1. Using [OpenSSL](https://www.openssl.org/) create a certificate for your application. You'll need to adjust the CN value to match the Ingress Host Name for your project.
+```shell
+openssl req -nodes -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 -subj "/CN=entando.apps-crc.testing"
+```
+You should see output similar to this:
+```shell
+Generating a RSA private key
+.....................................................................++++
+........................................................................................................................................................................................++++
+writing new private key to 'tls.key'
+-----
+```
+Now you can create a TLS Secret using the generated files.
+
+2. Go to `Workloads → Secrets → Create` and select `From YAML`. Use the following YAML as a starting point and then click `Create`.
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: entando-tls-secret
+data:
+  tls.key: ''
+  tls.crt: ''
+type: kubernetes.io/tls
+```
+3. Click on `Actions → Edit Secret` and use the ```Browse...``` buttons to upload the key and cert files.
+
+Now you'll create the `entando-ca-cert-secret` Secret, similar to what was done in Scenario 3 but this time using the self-signed certificate. 
+
+4. Go to `Workloads → Secrets → Create` and select `Key/value secret`
+5. Set the `Secret Name`, e.g. `entando-ca-cert-secret` 
+6. Set the `Key`, e.g. `tls.crt`
+7. Set the `Value` by clicking `Browse...` and loading the cert file from Step 1, e.g. `tls.crt`
+8. Click `Create`
+9. Next go to `Workloads → ConfigMaps` and update or create a ConfigMap named `entando-operator-config.` This is the ConfigMap used by the Operator to configure the deployments. You can [download the Entando Operator template](../devops/entando-operator.md#add-a-new-configmap) as a starting point. Set the "data/entando.ca.secret.name" and "data/entando.tls.secret.name" to match the names from above.
```

Comment:
Next go -> Next, go

arrow not backticked

`entando-operator-config.` -> `entando-operator-config`.